cmd/app_test.go

```diff
@@ -127,6 +128,7 @@ func listDirsFromRootNode(t *testing.T, dagServ format.DAGService, path string,
 	rootDir, err := uio.NewDirectoryFromNode(dagServ, rootNode)
 	assert.NoError(t, err)
 	links, err := rootDir.Links(ctx)
+	assert.NoError(t, err)
```

Comment:
@masih What tool did you use to identify those missing err checks?

Reply:
https://staticcheck.dev/